integration-tests/pipelines/konflux-e2e-tests.yaml

```diff
@@ -101,7 +101,9 @@ spec:
         - name: id
           value: $(context.pipelineRun.name)
         - name: tags
-          value: env=konflux,user=build-service
+          value: >
+            'owner=konflux-devprod@redhat.com,project=Konflux,created-by=integration-pipeline,
+            iac=mapt,cluster-name=$(context.pipelineRun.name),k8s-type=kind,component=build-service'
```

Comment:
Is it expected that we're setting cluster-name to the pipelineRun name?

What does `iac=mapt` mean?

Reply:
I see :+1: 

The cluster name and the IaC tool seems like an implementation detail of the `provision-kind-cluster` task. Would it make more sense to add those tags in the task itself? That way, if the implementation of the task changes, you won't have to go and update the tags for all the users of the task

Reply:
good point! I will update that Task